avocado/utils/asset.py

```diff
@@ -509,6 +551,27 @@ def parsed_name(self):
     def relative_dir(self):
         return os.path.join(self._get_relative_dir(), self.asset_name)
 
+    @classmethod
+    def remove_asset_by_path(cls, asset_path):
+        """Remove an asset and its checksum.
+
+        To be fixed: Due the current implementation limitation, this method
+        will not remove the metadata to avoid removing other asset metadata.
+
+        :param asset_path: full path of the asset file.
+        """
+        msg = "{} not found. Could not remove."
+        try:
+            os.remove(asset_path)
+        except FileNotFoundError:
+            LOG.debug(msg.format(asset_path))
```

Comment:
In these situations it's a good idea to preserve the original exception (even if we add custom information) because users will get a much clearer feeling of the root cause (as it's common across all Python based apps, and even the errors closely relate to say, libc errors).

So my suggestion is:

```python
msg = "Could not remove {}: {}"
...
except FileNotFoundError as e:
   LOG.debug(msg.format(asset_path, e))
```

Reply:
I agree with you here and I will include the original exception. But actually, I'm going to take a different approach if you don't mind. 

In fact, the log was not printing properly because was getting "avocado.test" logger. So I will let the exception be raised and catch it on the plugin itself, there I have access to LOG_UI.

Reply:
> I agree with you here and I will include the original exception. But actually, I'm going to take a different approach if you don't mind.
> 
> In fact, the log was not printing properly because was getting "avocado.test" logger. So I will let the exception be raised and catch it on the plugin itself, there I have access to LOG_UI.

I don't see the changes you proposed here.